src/components/TextListItem.tsx

```diff
@@ -34,87 +35,129 @@ type Props = {
   leftImage?: boolean | undefined;
   navigation: StackNavigationProp<Record<string, any>>;
   noSubtitle?: boolean | undefined;
+  rightImage?: boolean | undefined;
+  textListItemWithBigCard?: boolean | undefined;
```

Comment:
we are inside `TextListItem`, i guess we do not have the name duplicated again in the prop. the other props are also fine without that prefix. and the card is the size of a card, there is no need for "big". i suggest `withCard` here as boolean prop to indicate, that the text list has a card or not.

Reply:
Yes, we must avoid unnecessary name repetition.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/74e7eb7f2b529145ccbb1f97a902aa1981ad5341